otwcode/otwarchive
# AO3-5975 Display warning on proxies

#  Issue

https://otwarchive.atlassian.net/browse/AO3-5975

## Purpose

Puts a giant pink warning about security on any page that is not being access from a host that matches ArchiveConfig.APP_HOST. The warning has a "Dismiss Notice" button, and pressing the button will set a session cookie to hide the banner.

I've added previews to Jira.

## Testing Instructions

Maybe we can temporarily change ArchiveConfig.APP_HOST on staging and make sure the banner appears and can be dismissed?
